src/test/resources/log4j2.xml

```diff
@@ -1,13 +1,17 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="info">
+<Configuration packages="net.technolords.library.log4j2">
```

Comment:
Perhaps we should work with multiple configurations files. One default, which is the existing file suitable for unit testing, and one where we will capture the implementation as a reference configuration. Meaning to say that the latter can only work when our work is finished. In other words, I would like to see some logging while debugging or work...

We can invoke custom configuration files using a system property `log4j.configurationFile`. See also:
- https://logging.apache.org/log4j/2.x/manual/configuration.html

Also, when looking at the content of the XML file, I would like to repeat my comment on Slack:

> Two twings: combine MyCustomAppender and ElasticSearch, and RollingFile is not needed right?